assets/src/scss/layout/_navbar.scss

```diff
@@ -1,6 +1,6 @@
 $menu-height: 68px;
```

Comment:
Why do we have both `$menu-height` and `$menu-height-large`, if they're the same? 🤔

Reply:
I'd say legacy code :grimacing: We clearly need to normalize those variables, I'd be inclined to do it in a specific PR as I have no idea what the extent of these changes would be

Reply:
Fair enough!